Our version of GNU sed appears to truncate this file during config.status
processing unless there's a newline at the end of the file.  Presumably,
the "standard Linux" version of GNU sed does not.

XXX Not appropriate for upstream; but a bug against our version of sed
should be filed.

--- gnome-desktop-3.16.2/gnome-version.xml.in.in	2014-03-06 05:55:01.000000000 -0800
+++ gnome-desktop-3.16.2/gnome-version.xml.in.in	2015-07-06 20:08:46.097414197 -0700
@@ -5,4 +5,4 @@
  <micro>@GNOME_MICRO@</micro>
  <distributor>@GNOME_DISTRIBUTOR@</distributor>
  @GNOME_DATE_COMMENT_START@<date>@GNOME_DATE@</date>@GNOME_DATE_COMMENT_END@
-</gnome-version>
\ No newline at end of file
+</gnome-version>
